cmd/server-manager/views/pages/championships/index.html

```diff
@@ -65,7 +65,18 @@ <h2>Active</h2>
                             <a class="btn btn-sm btn-primary" href="/championship/{{ $championship.ID }}/export">Export</a>
 
                             {{ if $writeAccess }}
-                                <a class="btn btn-sm btn-warning" href="/championship/{{ $championship.ID }}/edit">Edit</a>
+                                <div class="btn-group custom-race-actions-btn-group" role="group" aria-label="Edit Championship">
+                                    <a class="btn btn-sm btn-warning custom-race-actions-btn-big" href="/championship/{{ $championship.ID }}/edit">Edit</a>
+
+                                    <button id="btnGroupDrop2" type="button" class="btn btn-warning btn-sm custom-race-actions-btn-small dropdown-toggle dropdown-toggle-split" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
```

Comment:
this id won't be unique? I think this is in the looped championship block? could it incorporate the championship id or something? I think the other references to it would also need updating (the dropdown menu below?)